packages/testing-helpers/src/scopedElementsWrapper.js

```diff
@@ -0,0 +1,64 @@
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { html, LitElement, TemplateResult } from 'lit-element';
+import { nothing } from 'lit-html';
+import { isIterable } from './lib.js';
+
+const transform = template => {
+  if (isIterable(template)) {
+    return [...template].map(v => transform(v));
+  }
+
+  if (template instanceof TemplateResult) {
+    return html(template.strings, ...template.values);
+  }
+
+  return template;
+};
+
+class ScopedElementsClass extends ScopedElementsMixin(LitElement) {
+  static get properties() {
+    return {
+      scopedElements: { type: Object },
+      template: { type: Object },
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.scopedElements = {};
+    this.template = nothing;
+  }
+
+  async firstUpdated() {
+    // @ts-ignore
```

Comment:
why?

Reply:
It's because `ScopedElementsMixin` doesn't have `firstUpdated` method and the linter is not inferring LitElement methods.

![Screenshot 2020-04-24 at 15 39 27](https://user-images.githubusercontent.com/329771/80218688-ca1b3b80-8641-11ea-8c99-fd8fbff6228f.png)

Reply:
we should fix the mixin definition then. will pm on slack